nipype/interfaces/freesurfer/utils.py

```diff
@@ -954,6 +954,63 @@ def _gen_outfilename(self):
         return name + ext + "_converted." + self.inputs.out_datatype
 
 
+class MRIsCombineInputSpec(FSTraitedSpec):
+    """
+    Uses Freesurfer's mris_convert to combine two surface files into one.
+    """
+    in_file1 = File(exists=True, mandatory=True, position=1,
+                    argstr='--combinesurfs %s',
+                    desc='File to be combined with in_file2')
+    in_file2 = File(exists=True, mandatory=True, position=2,
+                    argstr='%s',
+                    desc='File to be combined with in_file1')
+    out_file = File(argstr='%s', position=-1, genfile=True,
+                    mandatory=True,
+                    desc='Output filename. Combined surfaces from in_file1 and '
+                         'in_file2.')
+
+
+class MRIsCombineOutputSpec(TraitedSpec):
+    """
+    Uses Freesurfer's mris_convert to combine two surface files into one.
+    """
+    out_file = File(exists=True, desc='Output filename. Combined surfaces from '
+                                      'in_file1 and in_file2.')
+
+
+class MRIsCombine(FSCommand):
+    """
+    Uses Freesurfer's mris_convert to combine two surface files into one.
+
+    For complete details, see the `mris_convert Documentation.
+    <https://surfer.nmr.mgh.harvard.edu/fswiki/mris_convert>`_
+
+    Example
+    -------
+
+    >>> import nipype.interfaces.freesurfer as fs
+    >>> mris = fs.MRIsCombine()
+    >>> mris.inputs.in_file1 = 'lh.pial'
+    >>> mris.inputs.in_file2 = 'rh.pial'
+    >>> mris.inputs.out_file = 'out.stl'
+    >>> mris.cmdline  # doctest: +ALLOW_UNICODE
+    'mris_convert --combinesurfs lh.pial rh.pial out.stl'
+    >>> mris.run()  # doctest: +SKIP
+    """
+    _cmd = 'mris_convert'
+    input_spec = MRIsCombineInputSpec
+    output_spec = MRIsCombineOutputSpec
+
+    def _list_outputs(self):
+        outputs = self.output_spec().get()
+        if any(self.inputs.out_file.startswith(pre) for pre in ['lh.', 'rh.']):
+            outputs['out_file'] = self.inputs.out_file
+        else:
+            outputs['out_file'] = 'lh.' + self.inputs.out_file
```

Comment:
From what I've seen:
- `mris_convert --combinesurfs lh.pial rh.pial out.stl` creates lh.out.stl
- `mris_convert --combinesurfs rh.pial lh.pial out.stl` creates lh.out.stl
- `mris_convert --combinesurfs rh.pial rh.smoothwm out.stl` creates lh.out.stl
- `mris_convert --combinesurfs lh.pial rh.pial rh.out.stl` creates rh.out.stl

So it doesn't seem to matter what order you pass in the files or even if both files are from the right hemisphere. As long as the output file doesn't start with `lh.` or `rh.`, it will prepend `lh.` to the output filename.